src/components/Home/home.js

```diff
@@ -0,0 +1,62 @@
+import React, { Fragment, useState } from 'react';
+import { Link, Redirect } from 'react-router-dom';
+import './home.css';
+import Input from './Loginlist/input/input';
+import Label from './Loginlist/label/label';
+
+const Home = () => {
+  const [password, setPassword] = useState('');
+  const [passwordError, setPasswordError] = useState(false);
+
+  function handleChange(name) {
+    setPassword(name);
+  }
+
+  const consultToken = () => {
+    if (!password.length) {
+      setPasswordError(true);
+    } else {
+      localStorage.setItem('token', password);
+      setPasswordError(false);
+
+      window.location.href = 'http://localhost:3000/list-shopping';
```

Comment:
🔴 We need to change this. What's going to happen is that when we're in production, we'll get redirected to `localhost...` instead of `production-url.com/list-shopping`.

This should be `'/list-shopping` instead, or just `/list` per @arche4's comment,.

Reply:
Listo ! :+1: